docs/api/algorand/submit-transaction.mdx

```diff
@@ -0,0 +1,71 @@
+# SubmitTransaction
+
+Send funds from a [DelegatedWallet](#TODO), to another (regular or delegated) wallet.
+
+## Request Parameters
+
+|     Name    |                Type               | Description |
+| ----------- | --------------------------------- | ----------- |
+| `unsigned`  | `string[]`                        | The array of transactions (previously returned in the `noSign` array) that have not been signed. |
+| `signed`    | `string[]`                        | The array of transactions (previously returned in the `toSign` array) that have been signed by the user. |
+
+## Responses
+
+### Regular response
+
+|     Name    |         Type        | Description |
+| ----------- | ------------------- | ----------- | 
+| `txIds`     | `string[]`          | The array of transaction Ids of the transactions that have been submitted. |
+
+### Validation errors
+
+Returned when the parameters are invalid.
+
+|     Name    |                              Content                              |
+| ----------- | ----------------------------------------------------------------- |
+| Status code | 400                                                               |
+| `errorId`   | `INVALID_PARAMS`                                                  |
+| `message`   | `Validation errors in SubmitTransaction request.`                 | 
+| `errors`    | Array of strings containing additional informations.              |
+
+### Not issued by provider
```

Comment:
Should it be `####` header?

Reply:
Not quite, these are all different kinds of errors (they are not a subcategory of validation errors). However, if you think that it is not clear, I can find some alternative.